test/manifest-parser-tests.js

```diff
@@ -55,14 +56,31 @@ describe('Manifest Parser', function() {
     assert.equal(parsedManifest.value.background_color.value, undefined);
   });
 
+  describe('icon parsing', function() {
+    it('parses basic string', function() {
```

Comment:
add some intermediate variables here? It's not at all clear what you're testing once you get a level or two deep